Formats/Pauper/Archetypes/Burn.json

```
@@ -3,8 +3,16 @@
     "IncludeColorInName": false,
     "Conditions": [
         {
-            "Type": "InMainboard",
-            "Cards": ["Chain Lightning"]
+            "Type": "OneOrMoreInMainboard",
+            "Cards": ["Chain Lightning","Bump in the Night"]
```

Non intended change:```suggestion
            "Type": "InMainboard",
            "Cards": ["Chain Lightning"]
```